bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/netutils/WebSocketConnection.java

```diff
@@ -128,14 +174,24 @@ public void onMessage(Session session, String message) {
                 return;
             }
 
-            WebSocketMessageListener listener = listeners.get(getListenerId(changedMessage.r, changedMessage.id));
+            ResourceType resourceType = changedMessage.r;
+            String resourceId = changedMessage.id;
+
+            if (resourceType == ResourceType.SCENES) {
+                // scene recalls
+                resourceType = ResourceType.GROUPS;
+                resourceId = changedMessage.gid;
+            }
+
+            WebSocketMessageListener listener = listeners.get(getListenerId(resourceType, resourceId));
             if (listener == null) {
-                logger.trace(
+                logger.debug(
```

Comment:
IMO DEBUG is the correct level here. It provides more insight (which could be helpful for users) than standard logging INFO but not so verbose as TRACE that it can be used to find  bugs or missing implementations.

This one is maybe between DEBUG and TRACE, but the information can be useful for the user, not only for the developer trying to fix something, so I would prefer DEBUG.

Reply:
I just checked, and I still get spammed by this when enabling TRACE. If I understand it correctly, deCONZ sends updates for all devices? In this case maybe we shouldn't even log anything, or we could use TRACE to have this detailed information, but all we need to do is ignore events about devices we don't care about since we don't have things for them?

If we subscribe to events for specific devices of interest, then it seems we have a bug that should be fixed. In that case I could help tracking this, if you don't experience this problem?

Reply:
Yes, there is no filter for the websocket connection, we always receive everything. Changing the log level of course does not reduce the number of messages, it just moves them. 

Not logging anything at all might hide issues e.g. with thing handlers not properly registering themself as listeners. This could also be covered by tests (and in that case I would say we can omit logging). Better tests would be a good idea for a lot of the code in deconz, but that is obviously out of scope here.

Reply:
We agree that it just moves them, of course. 🙂 But there's many of them, so having them as DEBUG when this is usually expected and normal, clutters the DEBUG view.

Anyway, since this usually does not impact users, either way is fine, although I see no good reason to revert that log level change.